File: 6-robot/ssr_pkg/src/yao_node.cpp
#include <ros/ros.h>
#include <iostream>
using namespace std;
#include <std_msgs/String.h>

int main(int argc, char *argv[])
{
    ros::init(argc, argv, "yao_node");
    cout << "hello boy" << endl;
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("giegie_daiwo", 10);
    ros::Rate loop_rate(10);
    while (ros::ok())
    {
        cout << "I will sent mesg";
        std_msgs::String msg;
        msg.data = "please get in the car";
        pub.publish(msg);
        cout << endl;
        loop_rate.sleep();
    }
    return 0;
}

File: 6-robot/ssr_pkg/scripts/yao_node.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# vim:fenc=utf-8
import rospy
from std_msgs.msg import String

if __name__ == "__main__":
    rospy.init_node("yao_node")
    rospy.logwarn("yao_node is running")
    pub = rospy.Publisher("gie_gie_dai_wo", String, queue_size=10)
    rate = rospy.Rate(10)
    while not rospy.is_shutdown():
        rospy.logwarn("i will sent msgs")
        msg = String()
        msg.data = "please get in the car"
        pub.publish(msg)
        rate.sleep()
